src/pretix/api/views/order.py

```diff
@@ -572,6 +574,36 @@ def resend_link(self, request, **kwargs):
             status=status.HTTP_204_NO_CONTENT
         )
 
+    @action(detail=True, methods=['POST'])
+    def send_email(self, request, **kwargs):
+        order = self.get_object()
+        with language(order.locale, self.request.event.settings.region):
+            email_context = get_email_context(event=order.event, order=order)
+        email_subject = request.data.get('subject')
+        email_template = LazyI18nString(request.data.get('message'))
+        if not email_subject:
+            return Response({'detail': 'Subject is missing.'},
+                            status=status.HTTP_400_BAD_REQUEST)
+        if not email_template:
+            return Response({'detail': 'Message is missing.'},
+                            status=status.HTTP_400_BAD_REQUEST)
+        if not order.email:
+            return Response({'detail': 'There is no email address associated with this order.'},
+                            status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Instead of performing this validation manually, could you use a Django Rest Framework serializer to get consistent validation?